test/unit/tuple_uint32_overflow.c

```diff
@@ -0,0 +1,74 @@
+#include "tuple.h"
```

Comment:
Extract this test alongside with fix into separate patch.

Reply:
Extracted into separate commit.